src/world/area_kpa/kpa_81/A78510.c

```diff
@@ -4,9 +4,39 @@
 
 #include "world/common/AddPlayerHandsOffset.inc.c"
 
+extern s32 D_80240D98_A79258;
+extern s32 D_80240D9C_A7925C;
+
+// Needs data migrated
+#ifdef NON_MATCHING
+ApiStatus func_8024027C_A7873C(Evt* script, s32 isInitialCall) { 
+    s32* temp = script->ptrReadPos;
```

Comment:
`Bytecode* args` is how we name this (see shigure's PR I linked earlier - similar comments I made there)